exercises/leap/.meta/src/reference/java/Leap.java

```diff
@@ -0,0 +1,13 @@
+class Leap {
+
+    boolean isLeapYear(int year) {
+        return (
+            (
+                (year % 4 == 0) && 
+                (year % 100 != 0)
+            ) || 
+            (year % 400 == 0)
+            ); 
```

Comment:
There appears to be some issues with the formatting, this bracket should be further to the left I think.

Reply:
Fixed